lib/modules/dosomething/dosomething_api/resources/signup_resource.inc

```diff
@@ -34,26 +34,26 @@ function _signup_resource_definition() {
           'name' => 'resources/signup_resource',
         ],
         'callback' => '_signup_resource_index',
-        // 'args' => [
-        //   [
-        //     'name' => 'users',
-        //     'description' => 'The ids of specified users to get signups for.',
-        //     'optional' => TRUE,
-        //     'type' => 'string',
-        //     'source' => [
-        //       'param' => 'users'
-        //     ],
-        //     'default value' => NULL,
-        //   ],
-        //   [
-        //     'name' => 'campaigns',
-        //     'description' => 'The ids of the specified campaigns to get signups for.',
-        //     'optional' => TRUE,
-        //     'type' => 'string',
-        //     'source' => ['param' => 'campaigns'],
-        //     'default value' => NULL,
-        //   ],
-        // ],
+        'args' => [
+          [
+            'name' => 'user',
+            'description' => 'The id of the specified user to get signups for.',
+            'optional' => FALSE,
```

Comment:
this should be `TRUE`... as an index we should be able to call `/signups` and just get a paginated list of all signups. Granted it would be rare that we'd need to do that, but it shouldn't be mandatory to have to pass a user id.